thmlt.js

```diff
@@ -0,0 +1,10 @@
+let number=prompt("enter the number to find the odd number:")
```

Comment:
youre actually not using this number in your code at all. It should be telling you the odd numbers between the inputed number and 1.